mlx/ops.cpp

```diff
@@ -2905,4 +2901,24 @@ array tensordot(
   return reshape(matmul(x, y, s), rshape, s);
 }
 
+array outer(const array& a, const array& b, StreamOrDevice s /* = {} */) {
+  auto t_a = a;
+  if (a.ndim() > 0) {
+    t_a = flatten(a, s);
+  }
+  return multiply(reshape(t_a, {t_a.shape(0), 1}, s), flatten(b, s), s);
```

Comment:
```suggestion
  return multiply(reshape(a, {a.size(), 1}, s), flatten(b, s), s);
```

Reply:
had to do a static cast for this to compile